src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
-public class UserService {
+import core.basesyntax.exception.UserNotFoundException;
 
+public class UserService {
     public int getUserScore(String[] records, String email) {
-        return 0;
+
```

Comment:
this empty line is redundant

Reply:
There is no such line in my code on this page

Reply:
![image](https://user-images.githubusercontent.com/58385506/101164098-4e548900-363d-11eb-9ee1-6f8b5696fd1f.png)